src/content/app/species-selector/components/find-gene-instructions/FindGeneInstructions.tsx

```diff
@@ -0,0 +1,67 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import classNames from 'classnames';
+
+import { Step } from 'src/shared/components/step/Step';
+
+import SearchIcon from 'static/icons/icon_search.svg';
+
+import styles from './FindGeneInstructions.scss';
+import speciesLozengeStyles from 'src/shared/components/selected-species/SpeciesLozenge.scss';
+
+const FindGeneInstructions = () => {
+  return (
+    <section className={styles.instructionsPanel}>
+      <div className={styles.instructionsWrapper}>
+        <div className={styles.stepWrapper}>
+          <Step count={1} label="Find and add a species" />
+        </div>
+
+        <div className={styles.stepWrapper}>
+          <Step count={2} label="Select a species tab in any app">
+            <div className={styles.description}>
+              <div
+                className={classNames(
+                  speciesLozengeStyles.species,
+                  styles.speciesLozenge
+                )}
```

Comment:
Compare the relative sizes of 

1) the species lozenge in this component vs in the app bar,
2) the search icon

in the XD:

![image](https://user-images.githubusercontent.com/6834224/200783635-f240cade-aeb8-4ed1-b353-5e27a858637a.png)

and in the review deployment:

![image](https://user-images.githubusercontent.com/6834224/200783820-9ed58e70-b51b-4b1d-8722-700002fb388c.png)

Reply:
also the question mark size inside the search field

Reply:
I didn't make any change to this yet as it is outside the scope of this ticket. However, let's check with Andrea during the scrum tomorrow.

Reply:
> also the question mark size inside the search field

I didn't make any change to this yet as it is outside the scope of this ticket. However, let's check with Andrea during the scrum tomorrow.

However, I've made the other changes.